### PR TITLE
fix lib paths after Wandb 0.10

### PR DESCRIPTION
**Pull request by Borda**
## What does this PR do?

it seems that recently Horovod started failing without any touching...

## PR review    
Anyone in the community is free to review the PR once the tests have passed.     
If we didn't discuss your PR in Github issues there's a high chance it will not be merged.

## Did you have fun?
Make sure you had fun coding 🙃

### CODE REVIEW

**Comment by Borda** [comment]
@tgaddair mind help why the test is failing dues to missing PL?

**Comment by tgaddair** [comment]
@Borda can you explain the issue?

**Comment by Borda** [comment]
> @Borda can you explain the issue?

```
Wed Sep 16 15:12:50 2020[0]<stderr>:Traceback (most recent call last):
Wed Sep 16 15:12:50 2020[0]<stderr>:  File "/home/runner/work/pytorch-lightning/pytorch-lightning/tests/models/data/horovod/train_default_model.py", line 34, in <module>
Wed Sep 16 15:12:50 2020[0]<stderr>:    from pytorch_lightning import Trainer  # noqa: E402
Wed Sep 16 15:12:50 2020[0]<stderr>:ModuleNotFoundError: No module named 'pytorch_lightning'
Wed Sep 16 15:12:50 2020[1]<stderr>:Traceback (most recent call last):
Wed Sep 16 15:12:50 2020[1]<stderr>:  File "/home/runner/work/pytorch-lightning/pytorch-lightning/tests/models/data/horovod/train_default_model.py", line 34, in <module>
Wed Sep 16 15:12:50 2020[1]<stderr>:    from pytorch_lightning import Trainer  # noqa: E402
Wed Sep 16 15:12:50 2020[1]<stderr>:ModuleNotFoundError: No module named 'pytorch_lightning'
```
see: https://github.com/PyTorchLightning/pytorch-lightning/runs/1123823717

**Comment by tgaddair** [comment]
Thanks!  Looks like a syspath issue.  Does it only show up with Conda?

**Comment by Borda** [comment]
no, it shows for all when we install latest package versions, so the minimal versions are fine...
I have tried to freeze Horovod with `horovod!=0.20` but the problem is elsewhere...

**Comment by Borda** [comment]
it seems that the problem is in some package combination as this version was last passing (left) and actual is failing (right)
https://www.diffchecker.com/WKPtgbwn

File: requirements/extra.txt [hunk 1]
```diff
@@ -4,7 +4,7 @@ neptune-client>=0.4.109
 comet-ml>=3.1.12
 mlflow>=1.0.0
 test_tube>=0.7.5
-wandb>=0.8.21
+wandb>=0.8.21, <0.10.0  # higher version mess with system paths
```
Comment (tgaddair) [inline requirements/extra.txt:7]: Just a heads up, all versions do something to `sys.path`, but 0.10.0 adds even more stuff.

**Pull request merged**